Solutions/Citrix ADC/Data Connectors/Connector_CitrixADC_syslog.json

```diff
@@ -0,0 +1,135 @@
+{
+    "id": "CitrixADC",
+    "title": "Citrix ADC (former NetScaler)",
+    "publisher": "Citrix",
+    "descriptionMarkdown": "The [Citrix ADC (former NetScaler)](https://www.citrix.com/products/citrix-adc/) data connector provides the capability to ingest Citrix ADC logs into Microsoft Sentinel. If you want to ingest Citrix WAF logs into Microsoft Sentinel, refer this [documentation](https://learn.microsoft.com/azure/sentinel/data-connectors/citrix-waf-web-app-firewall)",
+    "additionalRequirementBanner": "These queries are dependent on a parser based on a Kusto Function deployed as part of the solution.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CitrixADC",
+            "baseQuery": "CitrixADCEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Event Types",
+            "query": "CitrixADCEvent\n | where isnotempty(EventType)\n    | summarize count() by EventType\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog",
+            "lastDataReceivedQuery": "CitrixADCEvent\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "CitrixADCEvent\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
```

Comment:
This query can be optimized like 
CitrixADCEvent\n   | where  TimeGenerated > ago(3d) \n| take 1 \n   | project IsConnected = true

Reply:
sure, will make the requested change and update.
Thanks